### DISCUSSION TITLE
Multi-language documents

### DISCUSSION THREAD

**Discussion by abersheeran** [NaturalSelectionLabs/RSS3-Protocol#26] [General]
如题，如果要做大，English 在目前来说是很好的起点，毕竟是事实上的互联网世界语，但其他语种也同样的重要，毕竟母语总是读起来更顺畅一些。

不如把联合国钦定的六种语言支持了？阿拉伯文、中文、英文、法文、俄文和西班牙文。这六种语言一出，基本能让互联网上 80% 以上的人都能看懂。

**Comment by usagiryu** [comment]
i18n 的事情，有必要做也一定会做，在合适的时间逐步做。我们也向广大社区募集志愿者，如果愿意去将 RSS3 翻译成各种语言，可以联系我们一起完善 RSS3 生态。

contact@rss3.io
